tsconfig.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Clean this up a bit? Don't need all the pre-generated comments.

Reply:
Removed!